packages/react-pdf-toolkit/src/components/icons/CheckCircleIcon/CheckCircleIcon.tsx

```diff
@@ -0,0 +1,4 @@
+import { Image } from '@/components/atoms/Image';
+import checkCircleIcon from '../../../assets/check-circle.png';
+
+export const CheckCircleIcon = () => <Image width={12} height={12} src={checkCircleIcon} />;
```

Comment:
Is it not possible to use an SVG or an icon package here over an image?

Reply:
P1

Reply:
PDF's supporting only png/jpg images.